react/hooks/useConfirmExit/index.js

```diff
@@ -142,6 +152,8 @@ export default function useConfirmExit({
       title={title}
       onCancel={onCloseModalRequest}
       onConfirm={onConfirm}
+      confirm={leave}
+      cancel={cancel}
```

Comment:
`cancel` or `confirm` are too generic and we don't know that they're string and used for primaryText / secondaryText. 

Instead we should have: 
`confirmLabel` and `cancelLabel`. 

but I'm not sure about creating a new API in fact. Why don't we use the existing Modal API with primaryText and secondaryText?

Reply:
and primaryAction / dismissAction. Since we have `Modal` in the component name, I expect the same API as our modal.

WDYT?

Reply:
I would prefer not.
In this flavour, they are not generic primary and secondary buttons. They have a dedicated functional role.

* We may put cancel as primary and confirm as secondary in the future (we hesitated yesterday)
* We do not want to let people think it can change the primary  and secondary role themselves because  it would mess with default labels.

Reply:
(I am changing the names for the labels)

Reply:
:+1: for confirmLabel and cancelLabel, I made the change